src/components/Match/matchColumns.jsx

```diff
@@ -46,168 +58,148 @@ export const heroTdColumn = {
   sortFn: true,
 };
 
+const parties = (row, match) => {
+  if (match.parties && Object.values(match.parties).reduce(sum) > 0) {
```

Comment:
You might want to add initial value.  I think it's always safer to have it unless we have a reason not to.  (I think this crashes if match.parties is `{}`)